modules/islandora_mirador/src/Plugin/Block/MiradorBlock.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Drupal\islandora_mirador\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides a mirador block from a IIIF Manifest.
+ *
+ * @Block(
+ *   id="mirador_block",
+ *   admin_label = @Translation("Mirador block")
+ * )
+ */
+class MiradorBlock extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultConfiguration() {
+    return [] + parent::defaultConfiguration();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function blockForm($form, FormStateInterface $form_state) {
+    $form['iiif_manifest_url'] = [
```

Comment:
Can you toss in the help text showing the user what tokens are available a la https://github.com/Islandora-CLAW/openseadragon/pull/14/files#diff-19f25ee42a8e255ead1cf86b6d673b22R29-R44

Reply:
ok, will do